internal/bufferedread/buffered_reader.go

```diff
@@ -103,6 +103,61 @@ func NewBufferedReader(object *gcs.MinObject, bucket gcs.Bucket, config *Buffere
 	return reader, nil
 }
 
+func (p *BufferedReader) prefetch() error {
+	// Do not schedule more blocks if the prefetch queue has reached its maximum configured size.
```

Comment:
```suggestion
	// Do not schedule more than MaxPrefetchBlockCnt.
```
Write the comment in terms of max number of block allowed (not config size)